common/components/ui/Modal.tsx

```diff
@@ -3,7 +3,7 @@ import React, { Component } from 'react';
 import './Modal.scss';
 
 export interface IButton {
-  text: string;
+  text: any;
```

Comment:
"Any" (😆) specific reason for this change?

Reply:
In order to take in `translate` output, it needed to take in React elements. Wasn't sure what the standard was, but I saw modal's `children` was `any` as well. Buttons should be able to have bold, italics, icons etc. so I figured this was better than forcing `translateRaw` everywhere.

Reply:
Does ```ts
string | React.ReactElement<string>
```
work?